GenderPayGap.WebUI/Views/Components/ReportOverview/ReportOverviewSections.cshtml

```diff
@@ -0,0 +1,43 @@
+﻿@using GenderPayGap.WebUI.Views.Components.ReportOverview
+@model GenderPayGap.WebUI.Views.Components.ReportOverview.ReportOverviewSectionsViewModel
+
+<h3 class="govuk-heading-m line">@(Model.Title)</h3>
```

Comment:
is there a reason why we don't add this as a caption? see here: https://design-system.service.gov.uk/components/table/

Reply:
should be a small heading

Reply:
This is because there are multiple tables on the page. So this wouldn't be a caption unless we made the page one big table, which would be a bit complicated for not much benefit.

I have however, made the heading of each section a caption i.e. "Person responsible", "Link to organisation" etc.